handlers/auth_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"crypto/rand"
 	"crypto/rsa"
 	"fmt"
+	"github.com/gaia-pipeline/gaia/helper/rolehelper"
```

Comment:
Please keep the following import path ordering:
- stdlib
- third party libraries
- gaia outside imports
- gaia internal imports

All divided by blank lines in between the sections. :)